dockerfiles/latest-trusty/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:14.04
+
+# all in one RUN to produce less, smaller layers
+RUN bash -c "\
+	sudo apt-get -qqy update; \
```

Comment:
Just curios why don't you just `RUN apt-get -qqy update && apt-get -qqy upgrade && etc.`